framework/include/cppmicroservices/Bundle.h

```diff
@@ -45,6 +45,8 @@ class ServiceReference;
 
 typedef ServiceReference<void> ServiceReferenceU;
 
+using clock_t = std::chrono::high_resolution_clock;
```

Comment:
Please stick to using `std::steady_clock`, because we want a monotonic clock here. If the high resolution clock was used to "fix" the failing MinGW BundleTest, increase the wait time in `BundleTest.cpp` from 10ms to 100ms instead.